### PR TITLE
Make Date dialog - Date Picker for Origin

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@africanmathsinitiative/developers this is ready for review. Fixes #4559

**Comment by maxwellfundi** [comment]
Reviewing this.

**Comment by maxwellfundi** [comment]
@dannyparsons  this is ready for you.

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -419,6 +421,10 @@ Public Class dlgMakeDate
         SetRCodeForControls(False)
     End Sub
 
+    Private Sub ucrDtpSpecifyOrigin_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrDtpSpecifyOrigin.ControlValueChanged
+        clsDateFunction.AddParameter("origin", Chr(34) & ucrDtpSpecifyOrigin.DateValue.Date & Chr(34), iPosition:=1)
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: The date picker is a user control like the others so you should be able to do this by setting the R code.

Comment (shadrackkibet) [reply instat/dlgMakeDate.vb]: We need an option for just returning a string e.g. "2018/01/01" which has not been implemented in the date time picker user control. Currently we have SetParameterIsRDate() which returns an RDate object. Once that is added i can now pass this through R code.,

Comment (dannyparsons) [reply instat/dlgMakeDate.vb]: Can this be completed now? We discuss that origin can be a Date object.

**Pull request merged**